src/actions/rotate/Rotate.ts

```diff
@@ -0,0 +1,47 @@
+import horizontalFlip from "./horizontalFlip";
+import verticalFlip from "./verticalFlip";
+import angle from "./angle";
+import Action from "../Action";
+import {IAction} from "../../interfaces/IAction";
+import Param from "../../parameters/Param";
+const HORIZONTAL_FLIP = 'hflip';
+const VERTICAL_FLIP = 'vflip';
+const PARAM_KEY = 'a';
+
+class Rotate extends Action implements IAction {
+  constructor(angle?: number) {
+    super();
+    this.addParam(new Param(PARAM_KEY, angle));
+  }
+
+  /**
+   * @return {Rotate}
+   */
+  horizontalFlip(): Rotate {
+    return this.addValue(HORIZONTAL_FLIP);
+  }
+
+  /**
+   * @return {Rotate}
+   */
+  verticalFlip(): Rotate {
+    return this.addValue(VERTICAL_FLIP);
```

Comment:
Add value - to which Param though?

An action can have multiple params

Reply:
it's private because it's a single param action